app/page.tsx

```diff
@@ -87,36 +111,94 @@ async function getMsgRolloutCollection(
 
 export default async function Dashboard() {
   // Check to see if Auth is enabled
-  const isAuthEnabled = process.env.IS_AUTH_ENABLED === 'true';
+  const isAuthEnabled = process.env.IS_AUTH_ENABLED === "true";
 
-  const recipeCollection = new NimbusRecipeCollection()
-  await recipeCollection.fetchRecipes()
-  console.log('recipeCollection.length = ', recipeCollection.recipes.length)
+  const recipeCollection = new NimbusRecipeCollection();
+  await recipeCollection.fetchRecipes();
+  console.log("recipeCollection.length = ", recipeCollection.recipes.length);
 
   // XXX await Promise.allSettled for all three loads concurrently
-  const localData = await getASRouterLocalMessageInfoFromFile()
-  const msgExpRecipeCollection = await getMsgExpRecipeCollection(recipeCollection)
-  const msgRolloutRecipeCollection = await getMsgRolloutCollection(recipeCollection)
+  const localData = await getASRouterLocalMessageInfoFromFile();
+  const msgExpRecipeCollection = await getMsgExpRecipeCollection(
+    recipeCollection
+  );
+  const msgRolloutRecipeCollection = await getMsgRolloutCollection(
+    recipeCollection
+  );
```

Comment:
Looks like all the above changes to `Dashboard` are just formatting changes.  We should get Prettier landed before long so that this stops happening. We can probably leave these changes in the PR; this not is just here for PR readers...

Reply:
Yeah, that sounds good. I can file a bug for landing Prettier